charts/prometheus-postgres-exporter/templates/serviceaccount.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.serviceAccount.create -}}
+{{- if and .Values.serviceAccount.create .Values.global.rbacEnabled -}}
```

Comment:
we are handling create inside helper templates if .Values.serviceAccount.create is false then default will be mapped
{{- if and .Values.serviceAccount.create .Values.global.rbacEnabled -}} -> {{- if .Values.global.rbacEnabled -}}

should be fine

Reply:
done